common/views/components/CardGrid/CardGrid.js

```diff
@@ -16,60 +18,89 @@ type ContentTypes = UiEvent | UiExhibition | Book | Article;
-              // $FlowFixMe
-              <ExhibitionPromo
-                id={item.id}
-                url={`/exhibitions/${item.id}`}
-                title={item.title}
-                format={item.format}
+              {item.type === 'exhibitions' && (
                 // TODO: (remove Picture type)
                 // $FlowFixMe
-                image={item.promoImage}
-                start={!item.isPermanent ? item.start : null}
-                end={!item.isPermanent ? item.end : null}
-                statusOverride={item.statusOverride}
-                position={i}
-              />
-            )}
-            {item.id !== 'tours' && item.type === 'events' && (
-              <EventPromo event={item} position={i} />
-            )}
-            {item.type === 'articles' && (
-              <StoryPromo
-                item={item}
-                position={i}
-                hidePromoText={hidePromoText}
-              />
-            )}
-            {item.type === 'books' && (
-              <BookPromo
-                url={`/books/${item.id}`}
-                title={item.title}
-                subtitle={item.subtitle}
-                description={item.promoText}
-                image={item.cover}
-              />
-            )}
-          </div>
-        ))}
+                <ExhibitionPromo
+                  id={item.id}
+                  url={`/exhibitions/${item.id}`}
+                  title={item.title}
+                  format={item.format}
+                  // TODO: (remove Picture type)
+                  // $FlowFixMe
+                  image={item.promoImage}
+                  start={!item.isPermanent ? item.start : null}
+                  end={!item.isPermanent ? item.end : null}
+                  statusOverride={item.statusOverride}
+                  position={i}
+                />
+              )}
+              {item.id !== 'tours' && item.type === 'events' && (
+                <EventPromo event={item} position={i} />
+              )}
+              {item.type === 'articles' && (
+                <StoryPromo
+                  item={item}
+                  position={i}
+                  hidePromoText={hidePromoText}
+                  hasTransparentBackground={itemsHaveTransparentBackground}
+                />
+              )}
+              {item.type === 'books' && (
+                <BookPromo
+                  url={`/books/${item.id}`}
+                  title={item.title}
+                  subtitle={item.subtitle}
+                  description={item.promoText}
+                  image={item.cover}
+                />
+              )}
+            </div>
+          ))}
+        </div>
       </div>
+      <Layout12>
+        <div
+          className={classNames({
+            // TODO: update with inter-component spacing when it's formalised
+            [spacing({ s: 3 }, { margin: ['top'] })]: true,
+          })}
+        >
+          {children}
+        </div>
+      </Layout12>
```

Comment:
Not sure why this can't be added to the component creating the `CardGrid?`
e.g. 
```JS
// page.js
<CardGrid />
<Layout12>
        <div
          className={classNames({
            // TODO: update with inter-component spacing when it's formalised
            [spacing({ s: 3 }, { margin: ['top'] })]: true,
          })}
        >
          {/* stuff */}
        </div>
      </Layout12>
```

It doesn't feel very specific to the `CardGrid` component?